service/tbc/tbc.go

```diff
@@ -817,6 +830,11 @@ func (s *Server) handleInv(ctx context.Context, p *peer, msg *wire.MsgInv) {
 	log.Tracef("handleInv (%v)", p)
 	defer log.Tracef("handleInv exit (%v)", p)
 
+	// XXX this currently does nothing. Blocks are requested elsewhere.
+	if true {
+		return
+	}
+
 	var bis []tbcd.BlockIdentifier
```

Comment:
This variable appears to only ever be written to (line 847) but never read from.
```
service/tbc/tbc.go:847:10: this result of append is never used, except maybe in other appends (SA4010)
```